### PR TITLE
WIP KET-284 branch for testing linkit d9 support upgrade

### PR DESCRIPTION
**Pull request by Matt-Hood**
Steps taken:

- Updated the composer.json file to point to a d9 ready branch 
- Tested for stability and regressions on the Kettering website 

Test instructions:
- Ensure the d9 compatible version of the module is successfully installed on Kettering 

Next steps:
- Check for regressions on all sites currently using duo_starter
Jira ticket:
[KET-284](https://codeandtheory.atlassian.net/secure/RapidBoard.jspa?rapidView=967&view=detail&selectedIssue=KET-284)

### CODE REVIEW

**Review by tekNorah — COMMENTED** [review]
I see your description indicates that this should point to a d9 ready branch?  But, looks like it is set to merge into the develop branch....

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
         "drupal/google_tag": "^1.1",
         "drupal/layout_builder_modal": "^1.0@alpha",
         "drupal/layout_builder_restrictions": "^2.2",
-        "drupal/linkit": "^4.3",
+        "drupal/linkit": "^6.0@beta",
```
Comment (tekNorah) [inline composer.json:29]: Per the Project page:
"The 8.x-5.x has removed support for link attributes. If you are in need for link attributes, please see D8 Editor Advanced link,"

Do you know if link attributes were being used?

If so, I would just go ahead and install the [Editor Advanced link](https://www.drupal.org/project/editor_advanced_link) [module]

Comment (Matt-Hood) [reply composer.json:29]: Thanks for pointing this out Norah, I will go ahead and double check on that

**Review by joshuaboltzmc — COMMENTED** [review]
@Matt-Hood I think we decided on simply removing the module from duo_starter:composer.json in favor of including the module and version we want in the kettering project's composer.json instead.

**Pull request closed without merging**